test/models/publisher_test.rb

```diff
@@ -17,6 +17,42 @@ class PublisherTest < ActiveSupport::TestCase
     Rails.application.secrets[:api_eyeshade_offline] = @prev_offline
   end
 
+  describe "#confirm_pending_email!" do
+    let(:publisher) { publishers(:default) }
+    let(:confirm_email) { nil }
+
+    describe "confirm_email" do
+      describe "pending_email" do
+        describe "!email" do
+          it "should set email to pending_email" do
```

Comment:
Set the email to the pending one, cleanup the pending email.
This case is the initial one, when an email was not completely set-up and the user is still "pending"